File: frontend/src/app/models/user-model.ts
export class UserModel {
    id?: number = 0;
    identification: number = 0;
    name: string = '';
    birth_date: string = '';

    constructor(identification: number, name: string, birth_date: string, id?: number) {
        this.id = id;
        this.identification = identification;
        this.birth_date = birth_date;
        this.name = name;
    }
}

File: backend/src/main/java/dev/backend/Models/UserModel.java
package dev.backend.Models;

import java.util.Date;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table
public class UserModel {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "identification", unique = true, nullable = false)
    private Long identification;
    
    private String name;
    private Date birth_date;
}